src/Illuminate/Support/Collection.php

```diff
@@ -91,6 +91,17 @@ public function collapse()
     }
 
     /**
+     * Concat the collection with the given items.
+     *
+     * @param  mixed  $items
+     * @return static
+     */
+    public function concat($items)
+    {
+        return new static($this->getArrayableItems($items) + $this->items);
```

Comment:
Because `array_merge()` will replace numeric indices and the result would look ike this:

`["Scratch Lotto",  "Powerball",  "Mega",  "Millions"] `